.github/workflows/python.yml

```
@@ -0,0 +1,33 @@
+name: Python
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  protogen:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup protobuf compiler
+        uses: arduino/setup-protoc@v3
+        with:
+          version: 26.1
```

Please wrap the version number in double quotation marks, like this:

```suggestion
          version: "26.1"
```

While YAML allows you to omit quotation marks for simple values like version numbers, it's considered best practice to always use quotation marks for string values. This ensures that the value is consistently treated as a string, preventing potential parsing issues that may arise in certain situations. For instance, without quotation marks, a YAML parser might interpret a version number like `26.10` as `26.1`, leading to unexpected behavior.